README.md

```diff
@@ -5,23 +5,30 @@ https://danshoup.github.io/sign-up-idiot
 
 (jnsert final screenshot of page?)
 
-
 # Live Link
+https://hidden-earth-06071.herokuapp.com/
 
-# Summary of Processes
-
-# Future Development 
+# Description
+Sign Up, Idiot! gives users the opportunity to create events, manage events, and sign up for events all from one profile. The user dashboard displays all of those components while also providing you with a history of your events. The website's functionality is simplistic enough so even the "idiots" of the world can easily volunteer and create events. 
```

Comment:
There needs to be a more thorough description of our application here. Look at the User Story Barry created for reference. If you need a template to work off of, I have one that I can slack you.